chart/templates/preflight.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
```

Comment:
Taking into account that we are not 100% sure that this is going to be the final implementation and could leave leftover on our clients... What do you think if we wrap all that content with a condition like in https://github.com/CartoDB/carto-selfhosted-helm/blob/d506900ada50e7685167d752f514c5da95476be3/chart/templates/router/ingress.yaml#L1 ? 

For example, we can use the same "replicated.enabled" condition like in the chart.

Reply:
perfect, I'll do that, the same for support-bundle